src/stp/plays/offensive/AttackingPass.cpp

```diff
@@ -162,48 +135,49 @@ bool AttackingPass::shouldRoleSkipEndTactic() { return false; }
-    } else {
-        stpInfos["passer"].setKickOrChip(KickOrChip::KICK);
-    }
-    // Passer
-    stpInfos["passer"].setPositionToShootAt(passingPosition);
-    stpInfos["passer"].setShotType(ShotType::TARGET);
-}
+/// Should become new play that receives ball
+//void AttackingPass::calculateInfoForPass(const world::ball::Ball* ball) noexcept {
+//    /// Recalculate pass positions if we did not shoot yet
+//        /// For the receive locations, divide the field up into grids where the passers should stand,
+//        /// and find the best locations in those grids
+//        receiverPositionRight = computations::PositionComputations::determineBestGoalShotLocation(gridRight, field, world);
+//        receiverPositionLeft = computations::PositionComputations::determineBestGoalShotLocation(gridLeft, field, world);
+//        std::vector<computations::PositionComputations::PositionEvaluation> positions = {receiverPositionLeft,receiverPositionRight};
+//
+//        Vector2 passLocation = computations::PassComputations::determineBestPosForPass(positions);
+//    /// Receiver should intercept when constraints are met
+//    if (ball->getVelocity().length() > control_constants::HAS_KICKED_ERROR_MARGIN) {
+//        receiverPositionLeft.position = Line(ball->getPos(), ball->getPos() + ball->getFilteredVelocity()).project(passingPosition);
+//    } else if (ball->getVelocity().length() > control_constants::HAS_KICKED_ERROR_MARGIN) {
+//        receiverPositionRight.position = Line(ball->getPos(), ball->getPos() + ball->getFilteredVelocity()).project(passingPosition);
+//    }
+//
+//    // Receiver
+//    stpInfos["receiver_left"].setPositionToMoveTo(receiverPositionLeft.position);
+//    stpInfos["receiver_right"].setPositionToMoveTo(receiverPositionRight.position);
+//
+//    // decide kick or chip
+//    auto passLine = Tube(ball->getPos(), passingPosition, control_constants::ROBOT_CLOSE_TO_POINT / 2);
+//    auto allBots = world->getWorld()->getRobotsNonOwning();
+//    // For all bots except passer and receivers, check if they are on the pass line, aka robot should chip
+//    if (std::any_of(allBots.begin(), allBots.end(), [&](const auto& bot) {
+//            if ((stpInfos["passer"].getRobot() && bot->getId() == stpInfos["passer"].getRobot()->get()->getId()) ||
+//                (stpInfos["receiver_left"].getRobot() && bot->getId() == stpInfos["receiver_left"].getRobot()->get()->getId()) ||
+//                (stpInfos["receiver_right"].getRobot() && bot->getId() == stpInfos["receiver_right"].getRobot()->get()->getId())) {
+//                return false;
+//            }
+//            return passLine.contains(bot->getPos());
+//        })) {
+//        stpInfos["passer"].setKickOrChip(KickOrChip::CHIP);
+//    } else {
+//        stpInfos["passer"].setKickOrChip(KickOrChip::KICK);
+//    }
+//    // Passer
+//    stpInfos["passer"].setPositionToShootAt(passingPosition);
+//    stpInfos["passer"].setShotType(ShotType::TARGET);
+//}
```

Comment:
can this be removed?

Reply:
Same as the other block, need to code later, but needs to be disabled to test